pennylane/__init__.py

```diff
@@ -133,6 +136,63 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, name: str):
+    """load(name, quantum_circuit_object)
+    Load a plugin :func:`~.load` which can then be used to convert objects of
+    quantum circuits from other frameworks.
+
+    Args:
+        quantum_circuit_object: the quantum circuit that will be converted
+            to a PennyLane template
+        name (str): the name of the plugin to convert from
+
+    Returns:
+        _function: the PennyLane template created from the quantum circuit
+            object
+    """
+
+    if name in plugin_converters:
+
+        # loads the plugin load function
+        plugin_converter = plugin_converters[name].load()
+
+        # calls the load function of the converter on the quantum circuit object
+        return plugin_converter(quantum_circuit_object)
+
+    raise ValueError(
+        "Converter does not exist. Make sure the required plugin is installed "
+        "and supports conversion."
+    )
+
+
+def from_qasm(quantum_circuit: str):
+    """from_qasm(quantum_circuit)
+    Loads quantum circuits defined in QASM by using the converter in the
+    PennyLane-Qiskit plugin.
+
+    Args:
+        quantum_circuit (str): the name for the QASM string
+
+    Returns:
+        _function: the PennyLane template created based on the QASM string
+    """
+    return load(quantum_circuit, name='qasm')
+
+
+def from_qasm_file(qasm_filename: str):
+    """from_qasm_file(qasm_filename)
+    Loads quantum circuits defined in QASM by using the converter in the
+    PennyLane-Qiskit plugin.
+
+    Args:
+        qasm_filename (str): the name for the QASM file
+
+    Returns:
+        _function: the PennyLane template created based on the QASM file
+    """
+    return load(qasm_filename, name='qasm_file')
```

Comment:
Should we also have a `from_qiskit()` function to mirror the `from_qasm` and `from_qasm_file`?

Reply:
Added it!